src/PlayerbotAI.cpp

```diff
@@ -4144,35 +4142,31 @@ bool PlayerbotAI::AllowActive(ActivityType activityType)
     if (sPlayerbotAIConfig->botActiveAlone <= 0)
         return false;
 
-    if (sPlayerbotAIConfig->botActiveAlone >= 100)
-        return true;
-
-    if (maxDiff > 1000)
-        return false;
-
-    uint32 mod = 100;
-
-    // if has real players - slow down continents without player
-    if (maxDiff > 100)
-        mod = 50;
-
-    if (maxDiff > 150)
-        mod = 25;
-
-    if (maxDiff > 200)
-        mod = 10;
-
-    if (maxDiff > 250)
-    {
-        if (Map* map = bot->GetMap())
-        {
-            if (map->GetEntry()->IsWorldMap())
+    // use the configured 'botActiveAlone 'percentages. Downscale the activity, within 
+    // reason ,with the intension to give the server some breathing space when needed.
+    uint32 mod = sPlayerbotAIConfig->botActiveAlone > 100 ? 100 : sPlayerbotAIConfig->botActiveAlone;
+    
+    if (sPlayerbotAIConfig->botActiveAloneAutoScale) {
+        uint32 maxDiff = sWorldUpdateTime.GetMaxUpdateTime();
```

Comment:
It may be better to use `GetMaxUpdateTimeOfCurrentTable` to get the diff of the most recent period. GetMaxUpdateTime will record the highest diff, which usually occurs when the server starts (the bot login). The diff at this time generally does not last long.
```
uint32 maxDiff = sWorldUpdateTime.GetMaxUpdateTimeOfCurrentTable();
```

Reply:
Thanks for the feedback, gonna have a try in the evening :)

Reply:
Confirmed and changed, the results are a little more when up and dowscaling.